typescript/infra/scripts/utils.ts

```diff
@@ -75,7 +75,8 @@ export function withContext<T>(args: yargs.Argv<T>) {
   return args
     .describe('context', 'deploy context')
     .coerce('context', assertContext)
-    .demandOption('context');
+    .demandOption('context')
+    .alias('c', 'context');
```

Comment:
This conflicts with one of the other options. You'll need to remove the alias from it. (It's chain down below)